yt_dlp/extractor/dumpert.py

```diff
@@ -10,7 +10,7 @@ class DumpertIE(InfoExtractor):
     _VALID_URL = r'''(?x)
         (?P<protocol>https?)://(?:(?:www|legacy)\.)?dumpert\.nl(?:
             /(?:mediabase|embed|item)/|
-            (?:/toppers|/latest|/?)\?selectedId=
+            .*\?selectedId=
```

Comment:
```suggestion
            [^#]+[?&]selectedId=
```

Isn't this more accurate?